actors/runtime/runtime.go

```diff
@@ -305,3 +268,16 @@ func (b *CBORBytes) UnmarshalCBOR(r io.Reader) error {
 	*b = c.Bytes()
 	return err
 }
+
+// Discard is a helper
+type Discard struct{}
```

Comment:
This is now an internal detail for actors, not about their interface to the VM. Note there are no more concrete types in this file.

Move to `builtin/shared.go` or similar.